scenes/game/enums.gd

```diff
@@ -136,6 +136,7 @@ enum EventType {
 	EventType_Strike_Miss,
 	EventType_Strike_ChooseToDiscard,
 	EventType_Strike_ChooseToDiscard_Info,
+	EventType_Strike_CleanupUI,
```

Comment:
This needs to be named Strike_Cleanup or Strike_Completed.

I know it's not super clean throughout the whole thing, but generally LocalGame should not "know" about UI.
I'm particularly bad about this with the strings though.